de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/DhlDeliveryOrderRepository.java

```diff
@@ -122,12 +128,35 @@ private DeliveryOrder toDeliveryOrderFromPO(final I_DHL_ShipmentOrderRequest req
 		final I_DHL_ShipmentOrder firstOrder = ordersPo.get(0);
 
 		final ImmutableList<DhlCustomDeliveryDataDetail> dhlCustomDeliveryDataDetail = ordersPo.stream()
-				.map(po -> DhlCustomDeliveryDataDetail.builder()
-						.packageId(po.getPackageId())
-						.awb(po.getawb())
-						.sequenceNumber(DhlSequenceNumber.of(po.getDHL_ShipmentOrder_ID()))
-						.pdfLabelData(po.getPdfLabelData())
-						.build())
+				.map(po -> {
+					//					DhlCustomsDocument customsDocument = null;
```

Comment:
please clean this up, or add a comment explaining the use of this commented out code